### ISSUE TITLE
[RFC]: [Implement analytics]

### ISSUE THREAD

**Issue by arhimede** [laminas/getlaminas.org#215]
We need to decide what service to use,  in order to track website traffic.
Current options are : 

1. [Fathom](https://usefathom.com/)  US infrastructure, 
2. [Plausible](https://plausible.io/)      European infrastructure

If there are other suggestions , please feel free to add it here

**Comment by arhimede** [comment]
PHP RFC
https://wiki.php.net/rfc/phpnet-analytics

**Comment by gsteel** [comment]
No comment - I've used both and they are largely similar - Would we exceed the hits limit for the cheapest Fathom account though?

**Comment by arhimede** [comment]
> No comment - I've used both and they are largely similar - Would we exceed the hits limit for the cheapest Fathom account though?

The PHP RFC is about implementing a hosted Matomo instance , which can be an option for us too

**Comment by froschdesign** [comment]
@gsteel 
@arhimede means a self-hosted instance of Matomo.

**Comment by gsteel** [comment]
Yes, I understood that, but self-hosting a matomo instance is _more_ work - using an off-the-shelf service would be a lot easier

**Comment by settermjd** [comment]
> We need to decide what service to use, in order to track website traffic. Current options are :
> 
> 1. [Fathom](https://usefathom.com/)  US infrastructure,
> 2. [Plausible](https://plausible.io/)      European infrastructure
> 
> If there are other suggestions , please feel free to add it here

I thought that Fathom was on Canadian-based infrastructure, more or less.
